crates/cli/src/subcommands/build.rs

```diff
@@ -30,12 +31,21 @@ pub fn cli() -> clap::Command {
                 .action(SetTrue)
                 .help("Builds the module using debug instead of release (intended to speed up local iteration, not recommended for CI)"),
         )
+        .arg(
+            Arg::new("wasm_opt")
+                .long("wasm-opt")
+                .value_parser(clap::builder::BoolishValueParser::new())
+                .default_value("true")
+                .conflicts_with("debug")
+                .help("Whether to optimize the compiled module with wasm-opt (default: true)"),
+        )
```

Comment:
I could also be persuaded to merge `--wasm-opt` and `--debug` into a `--profile=debug|release|release-no-wasm-opt` flag if that's preferred.